ci/deploy.sh

```diff
@@ -34,6 +29,16 @@ ssh-add ci/deploy.key
 git remote set-branches --add origin gh-pages output
 git fetch origin gh-pages:gh-pages output:output
 
+# Configure versioned webpage
+python build/webpage.py \
+  --checkout=gh-pages \
+  --version=$TRAVIS_COMMIT
+
+# Generate OpenTimestamps
```

Comment:
I want to make sure I understand the timestamping strategy.  We still timestamp in the same way without `--wait` because we want to CI deploy script to terminate quickly.  However, we now have the ability to manually update all of the OTS Timestamps whenever we want as you did in https://github.com/dhimmel/code-96/commit/8dae193309ede8d00e62131d50b5f51095ccfa83 without rewriting history.  After the upgrade, those timestamps are no longer dependent on the calendar service.

Is that correct?  Do we need to document the upgrade instructions somewhere?

Reply:
Yes your understanding is correct. `--wait` is not feasible for CI builds as it could take days IIRC. Will update documentation.